File: js_programs/number_of_digits.js
function numberOfDigits(num) {
  let n = Math.abs(num);

  var count = 0;
  if (n >= 0) {
    count++;
  }
  while (n / 10 >= 1) {
    n /= 10;
    count++;
  }
  return count;
}
console.log(numberOfDigits(26625632));

File: java_programs/Number_Of_Digits/Number_Of_Digits.java

public class Number_Of_Digits {

    public int numberInDigits(int num) {
        int count = 1;
        num = Math.abs(num);

        while ((num /= 10) >= 1) {
            count++;
        }
        return count;
    }

    public static void main(String[] args) {

        Number_Of_Digits myObj = new Number_Of_Digits();

        int num = 256321;

        System.out.println(myObj.numberInDigits(num));

    }

}
